frontend/pages/project.tsx

```diff
@@ -0,0 +1,21 @@
+import { GetServerSideProps } from 'next';
+
+const Project: React.FC<{ projects: string[] }> = ({ projects }) => {
+  return (
+    <>
+      <h1>Project</h1>
+      <p>{projects}</p>
```

Comment:
Projects is an array, html can't display arrays, you need to handle it manually for each object in the array